src/natcap/invest/utils.py

```diff
@@ -816,18 +822,18 @@ def _assert_vectors_equal(
             actual_geom_wkt = actual_geom.ExportToWkt()
             expected_geom_shapely = loads(expected_geom_wkt)
             actual_geom_shapely = loads(actual_geom_wkt)
-            # Try comparing geoms exactly equal allowing for different
-            # geometry ordering
-            geoms_equal = expected_geom_shapely.equals(actual_geom_shapely)
+            # confusingly named, `equals` compares numbers exactly but
+            # allows coords to be in a different order.
+            # `equals_exact` compares numbers with a tolerance but
+            # coords must be in the same order.
+            geoms_equal = (
+                expected_geom_shapely.equals(actual_geom_shapely) or
+                expected_geom_shapely.equals_exact(actual_geom_shapely, 1e-6))
```

Comment:
The tolerance argument is absolute not relative. This isn't ideal because coordinate values can span several orders of magnitude. But `shapely` has an [open PR](https://github.com/shapely/shapely/pull/1231) to normalize the coordinate values before comparing. I think `1e-6` will be good enough for us in the mean time.